common/src/jni/main/cpp/conscrypt/netutil.cc

```diff
@@ -40,44 +34,34 @@
 #endif  // !_WIN32
 
 namespace conscrypt {
+namespace netutil {
 
 /**
- * Network utility methods.
+ * Copied from libnativehelper NetworkUtilites.cpp
  */
-class NetworkUtil {
- public:
-    /**
-     * Copied from libnativehelper NetworkUtilites.cpp
-     */
-    static bool setBlocking(int fd, bool blocking) {
+bool setBlocking(int fd, bool blocking) {
```

Comment:
@davidben Right, I wasn't making it out-of-line because I took it out of a class.  I did so because I didn't think it made sense to be inlined.